profiles/services/staff_sso.py

```diff
@@ -105,7 +110,7 @@ def create_email(
 def associate_email(
     profile: StaffSSOProfile,
     email: Email,
-    type: str = EmailTypes.WORK.value,  # type: ignore
+    type: str = str(EmailTypes.WORK),
```

Comment:
Nice!

I wonder if something like this might work too:
```suggestion
    type: EmailTypes = EmailTypes.WORK,
```

Then you can call `str()` on it before putting it into the DB

Reply:
when I try that I get this error:
"Incompatible default for argument "type" (default has type "tuple[str, str]", argument has type "EmailTypes")"